src/main/java/calculator/controller/CalculateController.java

```diff
@@ -0,0 +1,37 @@
+package calculator.controller;
+
+import calculator.service.Calculator;
+import calculator.service.CheckDelimiterType;
+import calculator.service.DelimiterExtractor;
+import calculator.service.NumExtractor;
+import calculator.view.InputView;
+import calculator.view.OutputView;
+// 사용자가 애플리케이션을 실행하면 적절히 처리하기 위한 클래스
+public class CalculateController {
+
+    // 사용자가 애플리케이션을 실행하면 적절히 처리하는 메소드
+    public static void run(){
+        String userInput = InputView.getUserInput();
+
+        // 사용자가 아무것도 입력하지 않았다면 0 출력
+        if(userInput.isEmpty()){
+            OutputView.printOutput("0");
+        }
```

Comment:
```suggestion
        // 사용자가 아무것도 입력하지 않았다면 0 출력
        if (userInput.isEmpty()){
            OutputView.printOutput("0");
        }
```

공백도 코딩 컨벤션이다. 라는 1주차 미션 공통 피드백 부분에도 언급된 부분입니다! 

저는 프리코스 커뮤니티에서 `우테코 코드 스타일 적용 방법`을 보고 프로젝트에 적용했는데. 저장하면 자동으로 포맷이 맞춰지니 이런 실수를 줄일 수 있었어요. 한번 적용해보시면 도움이 될 것 같아요!